torch/_inductor/fx_passes/pad_mm.py

```diff
@@ -325,6 +326,17 @@ def should_pad_bench(
         if m_padded_length == k_padded_length == n_padded_length == 0:
```

Comment:
Should this condition be `any(x == 0 for x in (...))`.

Reply:
hmm, not sure I follow. what do you mean exactly ?

Reply:
The quoted code checks if all of m, n and k are 0. Your new code checks if any of the shape variables are 0. 

I think we could cover all cases with one check that any of m, n or k are zero.

Reply:
the quoted code is for the padded length difference, as opposed to the shapes themselves. the padded length we want to skip if they're all zero, whereas for the shapes themselves we want to skip if any are 0

Reply:
Oh I see, "padded length" actually means "length of padding" here. Confusing name, but preexisting so okay.

Reply:
agreed..